controllers/cloudflare_api.go

```diff
@@ -324,28 +247,16 @@ func (c *CloudflareAPI) validateTunnelId() bool {
 		return false
 	}
 
-	req, _ := http.NewRequest("GET", fmt.Sprintf("%saccounts/%s/tunnels/%s", CLOUDFLARE_ENDPOINT, c.ValidAccountId, url.QueryEscape(c.TunnelId)), nil)
-	if err := c.addAuthHeader(req, false); err != nil {
-		return false
-	}
-
-	client := &http.Client{}
-	resp, err := client.Do(req)
+	ctx := context.Background()
+	rc := cloudflare.AccountIdentifier(c.ValidAccountId)
+	tunnel, err := c.CloudflareClient.GetTunnel(ctx, rc, c.TunnelId)
 	if err != nil {
-		c.Log.Error(err, "error code in getting tunnel by Tunnel ID", "tunnelId", c.TunnelId)
+		c.Log.Error(err, "error retrieving tunnel")
```

Comment:
Any reason to remove the tunnel ID from the log? I thought it might be easier to tie what errors point to what tunnels since it would not have any identifier otherwise.
```suggestion
		c.Log.Error(err, "error retrieving tunnel", "tunnelId", c.TunnelId)
```